app/helpers/orchid/display_helper.rb

```diff
@@ -1,16 +1,21 @@
 module Orchid::DisplayHelper
-  # TODO refactor this?
-  # It will need to handle nested fields if nothing else
-  
-  def metadata(res, label, search_ele, link_bool=true)
+  # res: API result hash
+  # label: field display label ("Category")
+  # search_ele: the corresponding API field
```

Comment:
Is "ele" an abbreviation of "element"? Should we maybe change this to `search_field`? That would fit with the description comment better too. `api_field` or just `field` if you want it to be a shorter name :)

Reply:
sounds good to me!  I have a feeling this terminology is like 4 years old, since all this code came out of OSCYS back in the day, and I like the idea of rethinking it.  api_field seems the most straightforward to me!